src/util/ecoTypeToken.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@helix-foundation/contracts/currency/ECO.sol";
+import "@helix-foundation/currency/contracts/currency/ECO.sol";
```

Comment:
Changed the path and the remappings so that the import path is the same in our foundry & hardhat project

Reply:
(in hardhat it is required to start with `@helix-foundation/currency` since it's the npm package name)